src/components/Container/Container.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from "styled-components";
+import { HTMLAttributes } from "react";
+import { Orientation } from "@/components";
+
+type AlignItemsOptions = "start" | "center" | "end";
+type GapOptions = "none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl" | "xxl";
+type GrowthOptions = "0" | "1" | "2" | "3" | "4" | "5" | "6";
+type JustifyContentOptions =
+  | "flex-start"
```

Comment:
The start and flex-start are the same so may be we can remove it and add flex-start when start is passed

Reply:
Removed flex-start and flex-end.

Reply:
I would prefer if we simply had "start" and "end" instead of `flex-start` and `flex-end`